libraries/AP_Logger/LogFile.cpp

```diff
@@ -1078,3 +1078,18 @@ void AP_Logger::Write_OADijkstra(uint8_t state, uint8_t error_id, uint8_t curr_p
     };
     WriteBlock(&pkt, sizeof(pkt));
 }
+
+void AP_Logger::Write_SimpleAvoidance(uint8_t state, const Vector2f& desired_vel, const Vector2f& modified_vel, bool back_up)
+{
+    struct log_SimpleAvoid pkt{
+        LOG_PACKET_HEADER_INIT(LOG_SIMPLE_AVOID_MSG),
+        time_us         : AP_HAL::micros64(),
+        state           : state,
+        desired_vel_x   : desired_vel.x * 0.01f,
+        desired_vel_y   : desired_vel.y * 0.01f,
+        modified_vel_x  : modified_vel.x * 0.01f,
+        modified_vel_y  : modified_vel.y * 0.01f,
+        backing_up      : back_up,
+    };
+    WriteBlock(&pkt, sizeof(pkt));
+}
```

Comment:
need a extra line to get rid of the stop sign thingy

Reply:
I am sorry I did not understand this one... Can you please explain a little bit more?

Reply:
![image](https://user-images.githubusercontent.com/33176108/89032345-5c5f6980-d32c-11ea-90ae-c3bde8d60a1a.png)

Its good practice to leave a blank line at the end of the file, GitHub highlights this with a 'stop sign thingy'.

Reply:
Oh haha okay... Thanks! I will do that :) I did not know..

Reply:
annoyingly it only shows it in the file review tab not in the comment